### PR TITLE
1.86 cherrypicks

### PR DESCRIPTION
**Pull request by Duciwuci**
/kind enhancement

Please consider:
* [Fix initial VPN connection delay for SNA clusters](https://github.com/stackitcloud/gardener/pull/34) had merge conflicts :)

### CODE REVIEW

**Comment by Duciwuci** [comment]
/hold cancel

Cherries are now picked

**Comment by MichaelEischer** [comment]
We should also bump etcd-druid to 0.21.1 . The 1.86.4 includes that version in its imagevector

File: cmd/gardenlet/app/app.go [hunk 1]
```diff
@@ -435,6 +453,156 @@ func (g *garden) Start(ctx context.Context) error {
...
+			grmDeploy.Spec.Template.Spec.Volumes[grmCMVolumeIndex].ConfigMap.Name = newGRMConfigMap.Name
+
+			return seedClient.Patch(ctx, grmDeploy, patch)
+		})
+	}
+	return flow.Parallel(tasks...)(ctx)
+}
+
+// TODO(rfranzke): Remove this code after v1.83 has been released.
+func cleanupLegacyLoggingManagedResource(ctx context.Context, seedClient client.Client) error {
```
Comment (MichaelEischer) [inline cmd/gardenlet/app/app.go:583]: This function is no longer used. Please remove it.

File: VERSION
Comment (nschad) [inline VERSION:1]: Can you order this commit to be last in history?

**Review by nschad — CHANGES_REQUESTED** [review]

File: VERSION
Comment (nschad) [reply VERSION:1]: Referring to the 12ac6a384cb5e7a2486cdf99836df1e51de20c41 Commit. Just order it to be HEAD of the current branch.

**Comment by Duciwuci** [comment]
approve

**Pull request merged**